docker/docker-compose.yml

```diff
@@ -151,7 +151,7 @@ services:
 
   db:
     container_name: supabase-db
-    image: supabase/postgres:14.1.0
+    image: supabase/postgres:14.1.0.15
```

Comment:
cc @dragarcia - this one works right? It just has some bugs which are resolved in the latest version?